src/couch_mrview/include/couch_mrview.hrl

```diff
@@ -70,7 +70,9 @@
 
     direction = fwd,
     limit = ?MAX_VIEW_LIMIT,
+    max_view = ?MAX_VIEW_LIMIT,
     skip = 0,
+    skip_design = false,
```

Comment:
We are modifying the `#mrargs` record. I believe this record is passed between nodes. In that case, say if there is a life service ;-) which upgrades nodes in a rolling reboot fashion, the mis-matched `#mrargs` fields would cause view and view-like queries from old to new nodes to crash.

One option here could be to use the `extra` props field. It's not pretty but it should help for live upgrades. Eventually we should probably just switch to a map.

Reply:
Oops, I didn't realize `#mrargs` was passed between nodes. Agree that we shouldn't modify it like this due to upgrade incompatibilities. Maps sure would be nice here.